pkgs/by-name/pe/perses/package.nix

```
@@ -118,6 +119,12 @@ buildGoModule (finalAttrs: {
   passthru = {
     updateScript = ./update.sh;
 
+    tests.nixos = nixosTests.openssh.extendNixOS {
+      module = {
+        services.perses.package = finalAttrs.finalPackage;
+      };
+    };
+
```

I'm a bit confused by this. The openssh test doesn't set `services.perses.enable = true`, so this seems like it wouldn't have any effect?

Maybe adding the new test instead would be better?

```suggestion
    tests.nixos = nixosTests.perses;

```